reflect/src/main/java/dagger/reflect/Scope.java

```diff
@@ -155,7 +205,14 @@ Builder addBindingIntoSet(Key key, Binding elementBinding) {
         throw new IllegalArgumentException("key.type() must be Set");
       }
 
-      bindings.addIntoSet(key, elementBinding);
+      SetBindings bindings = setBindings.get(key);
```

Comment:
this `bindings` variable shadows the `Builder.bindings`
```suggestion
      SetBindings setBindings = setBindings.get(key);
```

Reply:
Your suggestion also shadows. It's weird to name the value from a map the same name as the map. The map names might need changed. Technically this one is like `setBindingses`.